tests/runner/test_parallel_runner.py

```diff
@@ -132,7 +132,7 @@ def test_max_worker_windows(self, mocker):
 @pytest.mark.parametrize("is_async", [False, True])
 class TestInvalidParallelRunner:
     def test_task_validation(self, is_async, fan_out_fan_in, catalog):
-        """ParallelRunner cannot serialize the lambda function."""
+        """ParallelRunner cannot serialise the lambda function."""
```

Comment:
```suggestion
        """``ParallelRunner`` cannot serialise the lambda function."""
```

Reply:
@AntonyMilneQB FYI I decided against doing this now, since it'll have to be applied across the whole file, and then may as well start looking through the whole codebase. In any case, it's a test, so the docstring is a bit less important?

Reply:
Yeah, that's totally fine. We don't have any real standards for docstrings of tests. They don't appear in the documentation at all so the argument for using double backtick doesn't really hold here.